SampleProject/Assets/Microsoft.Maps.Unity.Examples/CurrentLocationExample/Scripts/CurrentLocation.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+#if UNITY_WSA && !UNITY_EDITOR
+using Windows.Devices.Geolocation;
+#endif
+using TMPro;
+using Microsoft.Maps.Unity;
+using Microsoft.Geospatial;
+
+/// <summary>
+/// This class retrieves the current location of the user via the connected network and returns the latitude and longitude to the MapRenderer.
+/// It only works on the actual device.
+/// </summary>
+public class CurrentLocation : MonoBehaviour
+{
+    public MapRenderer mapRenderer;
+    
```

Comment:
The convention we are using is private field + [SerializeField] attribute, with underscore for the naming convention on the private fields, e.g.

    [SerializeFIeld]
    private MapRenderer _mapRenderer = null;

If you want a public accessor, use a property that is backed by the private fields... If you think code access isn't necessary though, don't worry about adding properties.